gradle/testing/defaults-tests.gradle

```
@@ -119,10 +119,12 @@ allprojects {
         jvmArgs '--illegal-access=deny'
       }
 
+      def loggingConfigFileProperty = objects.fileProperty().fileValue(file("${resources}/logging.properties"));
```

`file("${resources}/logging.properties")` is already non-lazy since at this point the file is already created. there is no value in converting it back to a non-lazy type.


I would suggest the following if `resources` is located in the project directory. If it is somewhere else there is a better way to get to it.
```suggestion
      def loggingConfigFileProperty = layout.projectDirectory.file("${resources}/logging.properties");
```

---

It's at `gradle/testing/defaults-tests/logging.properties`